src/rules/font_feature_values.rs

```diff
@@ -0,0 +1,128 @@
+//! The `@font-feature-values` rule.
+
+use cssparser::*;
+
+use super::Location;
+use crate::declaration::DeclarationBlock;
+use crate::error::{ParserError, PrinterError};
+use crate::parser::DefaultAtRule;
+use crate::printer::Printer;
+use crate::rules::CssRuleList;
+use crate::traits::{Parse, ToCss};
+use crate::values::ident::CustomIdent;
+use crate::values::string::CowArcStr;
+#[cfg(feature = "visitor")]
+use crate::visitor::Visit;
+
+/// A [@font-feature-values](https://drafts.csswg.org/css-fonts/#font-feature-values) rule.
+#[derive(Debug, PartialEq, Clone)]
+#[cfg_attr(feature = "visitor", derive(Visit))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "jsonschema", derive(schemars::JsonSchema))]
+#[cfg_attr(feature = "into_owned", derive(static_self::IntoOwned))]
+pub struct FontFeatureValuesRule<'i, R = DefaultAtRule> {
+  /// The name of the font feature values.
+  pub name: Vec<FontFeatureValuesRuleName<'i>>,
+  /// The declarations within the `@font-feature-values` rule.
+  #[cfg_attr(feature = "serde", serde(borrow))]
+  pub declarations: DeclarationBlock<'i>,
+  /// The rules within the `@font-feature-values` rule.
+  #[cfg_attr(feature = "serde", serde(default = "default_rule_list::<R>"))]
+  pub rules: CssRuleList<'i, R>,
```

Comment:
These rules are again more specific to `@font-feature-values` than any general at rule. There should be a type that's specific to the sub-rules of `@font-feature-values`. The `@page` rule provides an example of this.